airbyte-config/models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -424,7 +424,7 @@ private boolean getEnvOrDefault(final String key, final boolean defaultValue) {
     if (value != null && !value.isEmpty()) {
       return parser.apply(value);
     } else {
-      LOGGER.info("{} not found or empty, defaulting to {}", key, isSecret ? "*****" : defaultValue);
+      LOGGER.info("Default null or empty environment variable {} to '{}'", key, isSecret ? "*****" : defaultValue);
```

Comment:
```suggestion
      LOGGER.info("Using default value for environment variable {}: '{}'", key, isSecret ? "*****" : defaultValue);
```

"Default to" might be a little hard to read out of context